### DISCUSSION TITLE
Difference between this and Mac Mouse Fix?

### DISCUSSION THREAD

**Discussion by chris-c-thomas** [linearmouse/linearmouse#262] [General]
Just looking from a very brief overview I'm wondering what the main differences?

For the longest time I'd been using a Magic Keyboard + Magic Trackpad and still love that setup today. But I switched to a bluetooth mechanical from Keychron and MX Master 3S from Logitech which both support 3 devices (My MBP, Win 11 Desktop, and Fedora ThinkPad) seamlessly.

Mac Mouse Fix gave me a lot of functionality I was missing from not using the trackpade mainly with regards to gestures. Does Linear Mouse add more to the table for me?

**Comment by lujjjh** [comment]
This sort of utilities are all designed to make the mouse work better on macOS, but different people have different definitions of what "good" means.

Mac Mouse Fix focuses on making the mouse experience closer to the Magic Mouse and the trackpad, including smooth scrolling and a rich gesture system. If you are a heavy gesture user and are used to the acceleration curve of the mouse pointer on macOS, I'd highly recommend Mac Mouse Fix, right out of the box.

LinearMouse is more like an open source version of CursorSense + SteerMouse, focusing on customizing the mouse behavior. For example, you can adjust the speed and acceleration of the mouse pointer, set the number of lines per scroll to get a closer experience to Windows, or disable the mouse pointer acceleration for a better gaming experience.

Overall, although they both have some similar features, they are somewhat different in design philosophy and you can choose the one that suits you.

**Reply by chris-c-thomas** [reply]
I appreciate the response! I'm currently using the Magic Mouse Fix and I think that's what I've ultimately wanted. I do use lots of gestures having used the Magic Trackpad for so long that it was natural to have them available on my actual mouse now too.

**Comment by hzx505** [comment]
了解差异，准备体验 linearmause
